src/types.d.ts

```diff
@@ -17,7 +17,6 @@ type TrailingCommaOption =
 export interface PrettierConfig {
     printWidth?: number;
     tabWidth?: number;
-    useFlowParser?: boolean; // deprecated
     singleQuote?: boolean;
     trailingComma?: TrailingCommaOption;
     bracketSpacing?: boolean;
```

Comment:
All those settings are in fact not optional (they will default) . removing `?` will correct compiling issue (sorry, forgot to mention that earlier) @vvs